cli/src/test/java/com/devonfw/tools/ide/io/IdeProgressBarTest.java

```diff
@@ -1,42 +1,106 @@
-package com.devonfw.tools.ide.io;
-
-import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
-import static com.github.tomakehurst.wiremock.client.WireMock.any;
-import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
-import static com.github.tomakehurst.wiremock.client.WireMock.urlMatching;
-
-import java.nio.file.Path;
-
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.io.TempDir;
-
-import com.devonfw.tools.ide.context.AbstractIdeContextTest;
-import com.devonfw.tools.ide.context.IdeContext;
-import com.github.tomakehurst.wiremock.junit5.WireMockTest;
-
-/**
- * Test of {@link IdeProgressBar}.
- */
-@WireMockTest(httpPort = 8080)
-public class IdeProgressBarTest extends AbstractIdeContextTest {
-
-  /**
-   * Tests if a download of a file with a valid content length was displaying an {@link IdeProgressBar} properly.
-   *
-   * @param tempDir temporary directory to use.
-   */
-  @Test
-  public void testProgressBarDownloadWithValidContentLength(@TempDir Path tempDir) {
-
-    int maxLength = 10_000;
-
-    stubFor(any(urlMatching("/os/.*")).willReturn(aResponse().withStatus(200).withBody(new byte[maxLength])
-        .withHeader("Content-Length", String.valueOf(maxLength))));
-
-    IdeContext context = newContext(tempDir);
-    FileAccess impl = context.getFileAccess();
-    impl.download("http://localhost:8080/os/windows_x64_url.tgz", tempDir.resolve("windows_x64_url.tgz"));
-    assertThat(tempDir.resolve("windows_x64_url.tgz")).exists();
-    assertProgressBar(context, "Downloading", maxLength);
-  }
-}
+package com.devonfw.tools.ide.io;
+
+import com.devonfw.tools.ide.context.AbstractIdeContextTest;
+import com.devonfw.tools.ide.context.IdeContext;
+import com.devonfw.tools.ide.context.IdeTestContext;
+import com.devonfw.tools.ide.log.IdeLogLevel;
+import com.github.tomakehurst.wiremock.junit5.WireMockTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import java.nio.file.Path;
+
+import static com.devonfw.tools.ide.io.FileAccessImpl.defaultContentLength;
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.any;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlMatching;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * Test of {@link IdeProgressBar}.
+ */
+@WireMockTest(httpPort = 8080)

```

Comment:
This can be a new investigation task. I did not change this one, instead I added a constant at this place but the formatting is displaying this as a change.